mappings/net/minecraft/entity/ai/AiGoalTrackTarget.mapping

```diff
@@ -0,0 +1,32 @@
+CLASS xu net/minecraft/entity/ai/AiGoalTrackTarget
+	FIELD a checkCanNavigate Z
+	FIELD b canNavigateFlag I
+	FIELD c checkCanNavigateCooldown I
+	FIELD d timeWithoutVisibility I
+	FIELD e entity Lus;
+	FIELD f checkVisibility Z
+	FIELD g target Luk;
+	FIELD h maxTimeWithoutVisibility I
```

Comment:
A little verbose - ~`maxTimeNotVisible` or something similar?~

**Edit:** Probably not the best example, but I still think it's a little verbose, along with `timeWithoutVisibility` and `checkCanNavigateCooldown`.